src/monstergenerator.cpp

```diff
@@ -27,152 +27,151 @@
-    { "ELECTRIC", MF_ELECTRIC },
-    { "ACIDPROOF", MF_ACIDPROOF },
-    { "ACIDTRAIL", MF_ACIDTRAIL },
-    { "SHORTACIDTRAIL", MF_SHORTACIDTRAIL },
-    { "FIREPROOF", MF_FIREPROOF },
-    { "SLUDGEPROOF", MF_SLUDGEPROOF },
-    { "SLUDGETRAIL", MF_SLUDGETRAIL },
-    { "FIREY", MF_FIREY },
-    { "QUEEN", MF_QUEEN },
-    { "ELECTRONIC", MF_ELECTRONIC },
-    { "FUR", MF_FUR },
-    { "LEATHER", MF_LEATHER },
-    { "WOOL", MF_WOOL },
-    { "FEATHER", MF_FEATHER },
-    { "CBM_CIV", MF_CBM_CIV },
-    { "BONES", MF_BONES },
-    { "FAT", MF_FAT },
-    { "IMMOBILE", MF_IMMOBILE },
-    { "RIDEABLE_MECH", MF_RIDEABLE_MECH },
-    { "MILITARY_MECH", MF_MILITARY_MECH },
-    { "MECH_RECON_VISION", MF_MECH_RECON_VISION },
-    { "HIT_AND_RUN", MF_HIT_AND_RUN },
-    { "GUILT", MF_GUILT },
-    { "HUMAN", MF_HUMAN },
-    { "NO_BREATHE", MF_NO_BREATHE },
-    { "REGENERATES_50", MF_REGENERATES_50 },
-    { "REGENERATES_10", MF_REGENERATES_10 },
-    { "REGENERATES_1", MF_REGENERATES_1 },
-    { "REGENERATES_IN_DARK", MF_REGENERATES_IN_DARK },
-    { "FLAMMABLE", MF_FLAMMABLE },
-    { "REVIVES", MF_REVIVES },
-    { "CHITIN", MF_CHITIN },
-    { "VERMIN", MF_VERMIN },
-    { "NOGIB", MF_NOGIB },
-    { "ABSORBS", MF_ABSORBS },
-    { "ABSORBS_SPLITS", MF_ABSORBS_SPLITS },
-    { "LARVA", MF_LARVA },
-    { "ARTHROPOD_BLOOD", MF_ARTHROPOD_BLOOD },
-    { "ACID_BLOOD", MF_ACID_BLOOD },
-    { "BILE_BLOOD", MF_BILE_BLOOD },
-    { "REGEN_MORALE", MF_REGENMORALE },
-    { "CBM_POWER", MF_CBM_POWER },
-    { "CBM_SCI", MF_CBM_SCI },
-    { "CBM_OP", MF_CBM_OP },
-    { "CBM_TECH", MF_CBM_TECH },
-    { "CBM_SUBS", MF_CBM_SUBS },
-    { "FILTHY", MF_FILTHY },
-    { "SWARMS", MF_SWARMS },
-    { "CLIMBS", MF_CLIMBS },
-    { "GROUP_MORALE", MF_GROUP_MORALE },
-    { "INTERIOR_AMMO", MF_INTERIOR_AMMO },
-    { "NIGHT_INVISIBILITY", MF_NIGHT_INVISIBILITY },
-    { "REVIVES_HEALTHY", MF_REVIVES_HEALTHY },
-    { "NO_NECRO", MF_NO_NECRO },
-    { "PACIFIST", MF_PACIFIST },
-    { "PUSH_MON", MF_PUSH_MON },
-    { "PUSH_VEH", MF_PUSH_VEH },
-    { "PATH_AVOID_DANGER_1", MF_AVOID_DANGER_1 },
-    { "PATH_AVOID_DANGER_2", MF_AVOID_DANGER_2 },
-    { "PATH_AVOID_FALL", MF_AVOID_FALL },
-    { "PATH_AVOID_FIRE", MF_AVOID_FIRE },
-    { "PRIORITIZE_TARGETS", MF_PRIORITIZE_TARGETS },
-    { "NOT_HALLUCINATION", MF_NOT_HALLU },
-    { "CATFOOD", MF_CATFOOD },
-    { "CANPLAY", MF_CANPLAY },
-    { "CATTLEFODDER", MF_CATTLEFODDER },
-    { "BIRDFOOD", MF_BIRDFOOD },
-    { "PET_MOUNTABLE", MF_PET_MOUNTABLE },
-    { "DOGFOOD", MF_DOGFOOD },
-    { "MILKABLE", MF_MILKABLE },
-    { "NO_BREED", MF_NO_BREED },
-    { "PET_WONT_FOLLOW", MF_PET_WONT_FOLLOW },
-    { "DRIPS_NAPALM", MF_DRIPS_NAPALM },
-    { "DRIPS_GASOLINE", MF_DRIPS_GASOLINE },
-    { "ELECTRIC_FIELD", MF_ELECTRIC_FIELD },
-    { "STUN_IMMUNE", MF_STUN_IMMUNE },
-    { "LOUDMOVES", MF_LOUDMOVES },
-    { "DROPS_AMMO", MF_DROPS_AMMO }
-};
-
-} // namespace
-
 namespace io
 {
 
+// *INDENT-OFF*
```

Comment:
Nit: I'd limit the scope of this to switch lines only.
In missiondef.cpp too.

Reply:
I was of two minds about this.  Since you have expressed an opinion, I'll go with it.  (Also fixed `artifact.cpp`)